deny.toml

```diff
@@ -7,14 +7,14 @@ yanked = "deny"
 notice = "deny"
 ignore = [
     "RUSTSEC-2020-0056", # from gilrs v0.8.1 - unmaintained - https://github.com/koute/stdweb/issues/403
-    "RUSTSEC-2020-0095", # from crevice dev dependency - unmaintained - https://github.com/johannhof/difference.rs/issues/45
 ]
 
 [licenses]
 unlicensed = "deny"
 copyleft = "deny"
 allow = [
     "MIT",
+    "MIT-0",
```

Comment:
MIT-0 appears to be MIT with no attribution requirement. I don't think this will be a controversial addition at all, but any license addition is noteworthy and so... @cart ?

Reply:
I saw you guys already had `0BSD` which is essentially the same.

Reply:
Yeah this is pretty much what we're doing with Bevy (and what rust does). I'm cool with this.